client/client/client.go

```diff
@@ -1,21 +1,20 @@
 package client
 
 import (
-	"context"
+	"github.com/babylonlabs-io/babylon/client/babylonclient"
 	"time"
 
 	bbn "github.com/babylonlabs-io/babylon/app"
 	"github.com/babylonlabs-io/babylon/client/config"
 	"github.com/babylonlabs-io/babylon/client/query"
 	rpchttp "github.com/cometbft/cometbft/rpc/client/http"
-	"github.com/cosmos/relayer/v2/relayer/chains/cosmos"
 	"go.uber.org/zap"
 )
 
 type Client struct {
 	*query.QueryClient
 
-	provider *cosmos.CosmosProvider
+	provider *babylonclient.CosmosProvider
```

Comment:
Can we make this public? 
```suggestion
	Provider *babylonclient.CosmosProvider
```